croptalk/create_doc_data.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+from sqlalchemy import create_engine
+from dotenv import load_dotenv
+import os
+
+from croptalk.utils import query_sql_data
+
+load_dotenv("secrets/.env.secret")
+
+# Define your database connection URL
+db_url = os.environ["POSTGRES_URI"]
+
+# Create the SQLAlchemy engine
+engine = create_engine(db_url)
+
+if __name__ == '__main__':
+    # TODO this should be a SQL statement using the data from the generated weaviate_collection
+
+    # get chunks data, get unique s3 keys
+    chunks = pd.read_csv("data/total_chunks_in_weaviate_collection.csv")
```

Comment:
I don't think I see `data/total_chunks_in_weaviate_collection.csv` in the repo or PR... should it be added?

Reply:
I will actually create a db so we do not have to add it